hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieWriteMergeHandle.java

```diff
@@ -149,6 +158,7 @@ public HoodieWriteMergeHandle(HoodieWriteConfig config, String instantTime, Hood
   public HoodieWriteMergeHandle(HoodieWriteConfig config, String instantTime, String partitionPath,
                                 String fileId, HoodieTable<T, I, K, O> hoodieTable, TaskContextSupplier taskContextSupplier) {
     super(config, instantTime, partitionPath, fileId, hoodieTable, taskContextSupplier, true);
+    this.orderingFields = ConfigUtils.getOrderingFields(config.getProps());
```

Comment:
can we also initialize `readerContext` in this constructor and make the `readerContext` final? This constructor is for fg based handle, seems we can pass the `readerContext` in parameters.

Reply:
Unfortunately no it cannot be final, there is a setter method for the reader context